lib/fizzy/execute.cpp

```diff
@@ -27,6 +27,18 @@ void match_imported_functions(const std::vector<TypeIdx>& module_imported_types,
                                 " imported functions, " +
                                 std::to_string(imported_functions.size()) + " provided");
     }
+
+    for (size_t i = 0; i < imported_functions.size(); ++i)
+    {
+        const auto& module_imported_type = module_imported_types[i];
+        const auto& imported_type = imported_functions[i].type;
+        if (module_imported_type.inputs != imported_type.inputs ||
```

Comment:
It should be better to add `operator==` to `FuncType` to indicate that his type is comparable.

Reply:
Good idea, added